app-modules/form/src/Models/Submission.php

```diff
@@ -55,7 +55,7 @@ abstract public function submissible(): BelongsTo;
 
     abstract public function fields(): BelongsToMany;
 
-    public function author(): MorphTo
+    public function author(): MorphTo|BelongsTo
```

Comment:
```suggestion
    public function author(): BelongsTo
```

MorphTo extends BelongsTo